packages/settingregistry/src/settingregistry.ts

```diff
@@ -694,6 +720,8 @@ export class SettingRegistry implements ISettingRegistry {
       [phase in ISettingRegistry.IPlugin.Phase]: ISettingRegistry.IPlugin.Transform;
     };
   } = Object.create(null);
+  private _unloadedPlugins: { [name: string]: ISettingRegistry.IPlugin } =
```

Comment:
Would you mind using a Map rather than an object? It will clarify the meaning?

Reply:
Done in https://github.com/jupyterlab/jupyterlab/pull/14195/commits/3c03a7eb41349852b93e23e0b52ae479e5630c8a